test/AngularSpec.js

```diff
@@ -814,6 +814,19 @@ describe('angular', function() {
       expect(isElement(dst.jqObject)).toBeTruthy();
       expect(dst.jqObject.nodeName).toBeUndefined(); // i.e it is a jqLite/jQuery object
     });
+
+    it('should not merge the __proto__ property', function() {
+      var src = JSON.parse('{"__proto__": { "xxx": "polluted" } }');
```

Comment:
Nit: Inconsistent whitespace in the JSON string 😇

Reply:
Thanks!